### DISCUSSION TITLE
What's the difference between Orchard Core and Orchard CMS?

### DISCUSSION THREAD

**Discussion by billiejj** [OrchardCMS/OrchardCore#10299] [Q&A]
I'm a designer (not a developer) and a new client of mine IT dept is using Orchard CMS. They basically took my design and did a hack job putting it together in Orchard. The only thing that is templatized is the header and footer. Every other page is just an html block with code for the entire page. It looks like there are plenty of options to build a website in Orchard, but it refers to Orchard Core. I tried to load the design module and there isn't a module option and IT said it was because they just have Orchard CMS, rather than Core. They said they initially installed the template that comes with Orchard CMS but said it was a mess and deleted it. Do I ask them to update to Core rather than CMS? I'm having difficulty finding out the difference. It looks like Core is the most updated version of Orchard, but I'm not sure. Any help would be appreciated. I'm sure this is a considered a dirty word in this forum :-), but I am experienced with building templates in WordPress but usually use Dreamweaver to build sites.

**Comment by hishamco** [comment]
Orchard Core is a web framework built on top of ASP.NET Core, Orchard Core CMS is a Content Management System bult on top of Orchard Core, sometimes we refer to both as Orchard Core.

Backing to your question, seems they have Orchard CMS (Orchard 1) not Orchard Core, if is that please refer to https://github.com/OrchardCMS/Orchard

**Reply by billiejj** [reply]
So what the IT dept has installed is an outdated version (Orchard 1)? Why would someone use Orchard 1 instead of Orchard Core? I'm trying to figure out what I need to ask them to install, I already know that Orchard CMS is not going to work. The front end of the website is a simple informational website, but with many pages. With Orchard CMS installed, there is no benefit to designing the site in Orchard, I'd rather go back to Dreamweaver so I can build a proper site. What do I need to ask them to do so I can build template pages, widgets, etc.? Thank you for your help.

**Reply by billiejj** [reply]
and thank you!

**Reply by hishamco** [reply]
> I already know that Orchard CMS is not going to work

Why?

You can upgrade to Orchard Core, but this mean the development need to implemented to use the OC APIs

**Reply by Skrypt** [reply]
@billiejj Orchard Core version 1.0 has been launched recently. So, some companies don't trust a software if it's not released with a version 1.0 ... that's probably why they used Orchard CMS instead of Orchard Core.

Concerning the other questions, without saying that you are wrong, maybe since you are a designer you don't know all the reasons why this choice has been made. Maybe, you should ask them the reason(s) of their choice. Also, about using HTML pages simply, I don't think it is the best idea when it comes to manage things like url's, response cache, output cache ... just to name few things. Of course, you could use Liquid templates with Jekyll and serve the thing directly as it is. But, generally these design templates are only used by designers and integrated in a CMS afterward to be able to use their added functionalities ...

For example, in Orchard Core, you could use GraphQL in those HTML templates to query and display some data ... which is something that you can't do with a simple HTML wireframed template.

Also, basic feature which is really important to have in a website is generally to have the ability to send emails so it requires at least to use a server side function that will send these emails ... so a simple HTML website here won't be enough.

As for Orchard CMS it is actively maintained by the community, no more development is made on it though. So, using Orchard Core might be a better choice of course.

And, like @hishamco says, it will require to redo the integration work in OC but it should be fairly simple to do to be honest.

**Reply by billiejj** [reply]
I can't actually use anything they did for the front end so I suggested that they just delete what they have and install Orchard. Thanks for your feedback.
